packages/lit-dev-content/site/tutorials/content/svg-templates/04.md

```diff
@@ -0,0 +1,84 @@
+### Overview
+
+This is chapter of the demo covers the basics of patterns in SVG
+including how to:
+
+- create a `<pattern>` 
+- apply a `<pattern>` as a fill
+
+### Learn
+
+In SVG, the `<pattern>` element creates a tile that can be repeated. 
```

Comment:
I thought clipPath creates a tile? I'm getting confused, can you clarify here or in the last step?

Reply:
ah maybe "repeats the tile created by clip-path" doesn't matter

Reply:
changed it up